### DISCUSSION TITLE
Audio and video not synching

### DISCUSSION THREAD

**Discussion by jabolz** [mifi/lossless-cut#1842] [Support]
I have a 11GB mp4 video that I'm trying to cut commercial breaks from. I copy it into my working directory. I make all of the required cuts, export and merge and playback the finished file. Audio and video are not in synch. Also, the original mp4 is not in synch as well as the mp4 I copied it from. I have downloaded LosslessCut from both Microsoft Store and the latest from Github. Results are the same. BTW, how can you tell which version you are running. It's not in the about tab. Any ideas?

**Comment by mifi** [comment]
> Also, the original mp4 is not in synch as well as the mp4 I copied it from

Hi. Are you saying that the audio/video is not in sync in your original file that you're trying to cut using losslesscut, and when you cut/merge using LosslessCut, the exported file is still not in sync? If so, this is expected because losslesscut cannot automatically correct for sync issues. If you are asking for a way to shift the audio-track in order to make it in-sync, that is not yet possible but it's being tracked in this issue: #216

**Comment by jabolz** [comment]
No, the original mp4 is in sync. The copied mp4 is in sync. Only after the cut and merge are they not in sync -  Merged cut, original mp4 and copied mp4. Could this be a VLC issue? The PC I’m using is a BeeLink with Ryzen 7, 32MB RAM and Radeon 7840 running Windows 11 Pro. Plenty of horsepower and up-to-date OS.

**Reply by mifi** [reply]
Ok, I'm not sure what you mean by "the original mp4 is not in synch" but I assume that the original file has correct audio/video sync then.

What I would try first is to change parameters as suggested in [Common / known issues & troubleshooting](https://github.com/mifi/lossless-cut/blob/master/issues.md#common--known-issues--troubleshooting) and see if it makes a difference. Also try to cut *without merging*, and see if the audio is in sync in the individual outputted files

**Comment by jabolz** [comment]
OK, this time I took an mp4 of 8.3 GB and produced 9 cuts. I exported and merged. Only five of the nine cuts merged. I then merged and separated. Only five of the cuts separated. Any ideas?

**Comment by jabolz** [comment]
Kind of appears that Lossless-cut doesn’t like large files. I took the 8.2 GB mp4 and ran it thru Handbrake and reduced the size to 3.2. Ran Lossless-cut again with no issues. Also, can you tell me where to find in Loss-less cut the version I’m running.

**Comment by mifi** [comment]
did you try only cut without merge and see if the segments are fine?

**Comment by jabolz** [comment]
Yes, segments were out of synch. Used AVIDemux to resynch merged segments. They were about 3100 ms out od synch.  BTW, Lossless-cut download from Windows Store does not have version number in “help about’. Have been using 3.59.1.0 from GitHub which appears to work fine.

**Reply by mifi** [reply]
Unfortunately I don't know how to fix this, but if you're having the out-of-sync problem even when trying all suggestions from "troubleshooting", then it sounds like a bug in ffmpeg (which is used to cut files).

> BTW, Lossless-cut download from Windows Store does not have version number in “help about

Windows and Apple App Store runs on a different version range than github (although it's the same code). therefore versions have been disabled in the help menu. I believe Windows Store should be able to tell you which Windows store version of LosslessCut you're on..
